File: ass24/5.py
# 5. Write a program which accept string from user and count number of white spaces
# Input : “MarvellouS”
# Output : 0
# Input : “MarvellouS Infosystems”
# Output : 1
# Input : “MarvellouS Infosystems by Piyush Manohar Khairnnar”
# Output : 5

def CountWhite(s1):
    count = 0
    for i in s1:
        if(i == ' '):
            count = count + 1

    print("number of white spaces : ",count)

def main():
    print("Enter String :")
    str1 = input()

    CountWhite(str1)


if __name__ == '__main__':
    main()

File: ass26/5.js
// 5. Write a program which 2 strings from user and concat second string
// after first string. (Implement strcat() function).
// Input : “Marvellous Infosystems”
// “Logic Building”
// Output : “Marvellous Infosystems Logic Building”


function strcat(str1,str2)
{
    let lower ='';
    // for(var i=0; i<str.length; i++)
    // {
    //     if(str[i] == str[i].toLowerCase())
    //     {
    //         lower +=str[i]
    //     }
    // }
    return str1.concat(str2);

}
console.log(strcat('Marvellous Infosystems','Logic Building'))